staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3642,6 +3642,9 @@ func DescribeEvents(el *corev1.EventList, w PrefixWriter) {
 			interval = fmt.Sprintf("%s (x%d over %s)", translateTimestampSince(e.LastTimestamp), e.Count, translateTimestampSince(e.FirstTimestamp))
 		} else {
 			interval = translateTimestampSince(e.FirstTimestamp)
+			if e.FirstTimestamp.IsZero() {
+				interval = translateTimestampSince(metav1.Time(e.EventTime))
```

Comment:
`EventTime` is `metav1.MicroTime` but not `metav1.Time`. So you need another `translateMicroTimestampSince `. Like https://github.com/kubernetes/kubernetes/pull/89999

Reply:
done

Reply:
done